<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>UAVInstance</class>
  <widget class="QWidget" name="UAVInstance">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>30</height>
      </rect>
    </property>
    <property name="minimumSize">
      <size>
        <width>0</width>
        <height>0</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Form</string>
    </property>

    <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
        <number>0</number>
      </property>
      <property name="spacing">
        <number>0</number>
      </property>

      <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
          <property name="enabled">
            <bool>true</bool>
          </property>

          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,100,10,5,1">
            <property name="spacing">
              <number>2</number>
            </property>
            <property name="margin">
              <number>4</number>
            </property>
            <item>
              <widget class="QLabel" name="uavName">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>ID</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="xPos">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>x</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="yPos">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>y</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="zPos">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>z</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="xSpeed">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>vx</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="ySpeed">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>vy</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="zSpeed">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>vz</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="lat">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>lat</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="lon">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>lon</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="alt">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>alt</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="pitch">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>pitch</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="roll">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>roll</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="yaw">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>yaw</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="pitchSpeed">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>v_pitch</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="rollSpeed">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>v_roll</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="yawSpeed">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>25</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="lineWidth">
                  <number>1</number>
                </property>
                <property name="text">
                  <string>v_yaw</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>

  </widget>
  <resources />
  <connections />
</ui>